src/backend/LamaAPI/Infrastructure/Migrations/20200223102258_AddSearchHistory.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Infrastructure.Migrations
+{
+    public partial class AddSearchHistory : Migration
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=5053510)